extensions/src/libadalang-implementation-extensions.adb

```diff
@@ -50,17 +50,22 @@ package body Libadalang.Implementation.Extensions is
    -------------------------
 
    function Ada_Node_P_Get_Unit
-     (Node           : Bare_Ada_Node;
-      Name           : Symbol_Type_Array_Access;
-      Kind           : Analysis_Unit_Kind;
-      Load_If_Needed : Boolean) return Internal_Unit is
+     (Node               : Bare_Ada_Node;
+      Name               : Symbol_Type_Array_Access;
+      Kind               : Analysis_Unit_Kind;
+      Load_If_Needed     : Boolean;
+      Not_Found_Is_Error : Boolean;
+      Process_Parents    : Boolean := True) return Internal_Unit
+   is
    begin
       return Env_Hooks.Fetch_Unit
         (Node.Unit.Context,
          Env_Hooks.Symbol_Type_Array (Name.Items),
          Node.Unit,
          Kind,
-         Load_If_Needed);
+         Load_If_Needed,
+         Not_Found_Is_Error => Not_Found_Is_Error,
+         Process_Parents    => Process_Parents);
```

Comment:
I suppose you pass boolean arguments by named association to make sure the right boolean actual is passed to the right formal. In that case, can you add name assoc for `Load_If_Needed`?